healthcareai/common/filters.py

```diff
@@ -1,6 +1,25 @@
 import pandas as pd
+from sklearn.preprocessing import StandardScaler
 
 def remove_datetime_columns(df):
     # TODO: make this work with col names shorter than three letters
     cols = [c for c in df.columns if c[-3:] != 'DTS']
-    return df[cols]
\ No newline at end of file
+    return df[cols]
+
```

Comment:
Would you mind running pylinter on this? It'd catch things like spaces between arguments, etc